### PR TITLE
Descriptions for schematics.

### PR DESCRIPTION
**Pull request by summetdev**
If this PR gets merged, I will likely make a PR for CoreBot to view these descriptions in the #schematics channel.

preview

### CODE REVIEW

**Comment by Quezler** [comment]
will one line be enough? (why not e.g. the same size as the map description field in the editor?)

**Comment by MEEPofFaith** [comment]
🐱 is🐸

**Comment by unjown** [comment]
for this is not really necessary :)

**Comment by summetdev** [comment]
@Quezler done.
image

**Comment by stuffyAI** [comment]
Tough there weren't any suggestions regarding this you should be fine marking this ready for review, looking nice and good

**Comment by DeltaNedas** [comment]
anuke

File: core/src/mindustry/input/InputHandler.java [hunk 1]
```diff
@@ -499,11 +500,40 @@ protected void showSchematicSave(){
...
                 lastSchematic.tags.put("name", text);
+                lastSchematic.tags.put("description", description);
                 schematics.add(lastSchematic);
                 ui.showInfoFade("@schematic.saved");
                 ui.schematics.showInfo(lastSchematic);
             }
-        });
+        };
+
+        new Dialog("@schematic.add"){{
```
Comment (Anuken) [inline core/src/mindustry/input/InputHandler.java:510]: Is it *really* important for the player to get a description entry dialog when saving the schematic for the first time? If a player cares, they can add it in later. Could be annoying to have an extra field pop up every time you want to save a small utility schematic.

Comment (summetdev) [reply core/src/mindustry/input/InputHandler.java:510]: Yes, you are right. I'll fix it soon.

**Comment by summetdev** [comment]
@Anuken fixed.

**Pull request merged**